Assets/BossRoom/Scripts/Server/DebugCheatsManager.cs

```diff
@@ -0,0 +1,67 @@
+using Unity.Netcode;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+#if UNITY_EDITOR || DEVELOPMENT_BUILD
+namespace Unity.Multiplayer.Samples.BossRoom.Server
+{
+    /// <summary>
+    /// Handles debug cheat events, applies them on the server and logs them on all clients. This class is only
+    /// available in the editor or for development builds
+    /// </summary>
+    public class DebugCheatsManager : NetworkBehaviour
+    {
+        [SerializeField]
+        [Tooltip("Enemy to spawn. Make sure this is included in the NetworkManager's list of prefabs!")]
+        NetworkObject m_EnemyPrefab;
+
+        [SerializeField]
+        [Tooltip("Boss to spawn. Make sure this is included in the NetworkManager's list of prefabs!")]
+        NetworkObject m_BossPrefab;
+
+        public void SpawnEnemy()
+        {
+            SpawnEnemyServerRpc();
+        }
+
+        public void SpawnBoss()
+        {
+            SpawnBossServerRpc();
+        }
+
+        public void GoToPostGame()
+        {
+            GoToPostGameServerRpc();
+        }
```

Comment:
do you need those wrappers? Couldn't you call the RPCs themselves?

Reply:
The `UnityEventGameEventListener`s uses parameterless `UnityEvent`s, so I could not use the RPC calls directly

Reply:
I see, that's how C# method signature works :( 
nvm